code/modules/projectiles/ammo_boxes/misc_boxes.dm

```diff
@@ -138,3 +138,31 @@
 
 /obj/item/ammo_box/magazine/misc/flares/empty
 	empty = TRUE
+
+//------------------------Flashlight Box--------------------------
+
+/obj/item/ammo_box/magazine/misc/flashlight
+	name = "box of flashlights"
+	desc = "A box of flashlights to brighten your day!"
+	magazine_type = /obj/item/device/flashlight
+	num_of_magazines = 8
+	icon_state = "flashlightbox"
+	icon_state_deployed = "flashlightbox_deployed"
+	overlay_content = "_flashlight"
+
+/obj/item/ammo_box/magazine/misc/flashlight/empty
+	empty = TRUE
+
+//------------------------Battery Box--------------------------
+
+/obj/item/ammo_box/magazine/misc/power_cell
+	name = "\improper box of High-Capacity Power Cells"
```

Comment:
no need for improper here

Reply:
All the other boxes are /improper though... Actually I forgot to add the improper for the flashlights...